### DISCUSSION TITLE
facebook::react::recoverable: could not open filesuper.getJsbundleFile

### DISCUSSION THREAD

**Discussion by thepunisherxx** [pavjacko/renative#1103] [General]
Title: Issue with facebook::react::recoverable - Unable to open filesuper.getJsbundleFile

Description:

Hello renative team,

I've encountered an issue while working with the React Native integration for Facebook. The specific error message I'm seeing is:

arduino

`facebook::react::recoverable: could not open filesuper.getJsbundleFile`

Environment:

    Renative Version: 0.37.3
    React Native Version: 0.67.2
    Platform: android / androidtv

Steps to Reproduce:

    Install Renative with flexn create template starter

Actual Behavior:
Instead, I received the aforementioned error message, andthe screen goes blank
![Screenshot 2023-09-12 194610](https://github.com/flexn-io/renative/assets/88408465/d6b2059d-8e78-4421-9c5e-5fcf889bf995)

I'd appreciate any guidance or assistance in resolving this issue. Thank you for your time and support!

**Comment by pauliusguzas** [comment]
@wwwebgainss0 thanks for reporting this, what cli command are you running?

**Reply by thepunisherxx** [reply]
I didn't see it, sorry. :( 

`rnv export -p android`

**Reply by pauliusguzas** [reply]
rnv export command only creates build but does not install it, and in the original message you showed emulator screenshot. Are you sure this was the command? If so then it is expexted that nothing happens on emulator and that error might be from other unrelated app
